chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/MeaningfulUseController.java

```diff
@@ -0,0 +1,185 @@
+						muu.setError("Line " + muu.getCsvLineNumber() + ": Field \"num_meaningful_use\" with value \"" + currRecord.get(1).trim() + "\" is invalid. "
+								+ "Value in field \"num_meaningful_use\" must be an integer.");
+						muusToUpdate.add(muu);
+						uniqueMuusFromFile.add(muu.getProductNumber());
+					}
+					catch (IOException e){
+						muu.setProductNumber(chplProductNumber);
+						muu.setCsvLineNumber(i);
+						Integer dupLineNumber = null;
+						// get line number with duplicate chpl_product_number
+						for (MeaningfulUseUser entry: muusToUpdate) {
+						     if (entry.getProductNumber().equals(muu.getProductNumber())){
+						    	 dupLineNumber = entry.getCsvLineNumber();
+						     }
+						   }
+						muu.setError("Line " + muu.getCsvLineNumber() + ": Field \"chpl_product_number\" with value \"" + muu.getProductNumber() + "\" is invalid. "
+								+ "Duplicate \"chpl_product_number\" at line " + dupLineNumber);
+						muusToUpdate.add(muu);
+					}
+				}
+			}
+		} catch(IOException ioEx) {
+			logger.error("Could not get input stream for uploaded file " + file.getName());			
+			throw new ValidationException("Could not get input stream for uploaded file " + file.getName());
+		} finally {
+			 try { parser.close(); } catch(Exception ignore) {}
+			try { reader.close(); } catch(Exception ignore) {}
+		}
+		
+		try {
+			meaningfulUseUserResults = cpManager.updateMeaningfulUseUsers(muusToUpdate);
+		} catch (JsonProcessingException e) {
+			e.printStackTrace();
+		} catch (EntityCreationException e) {
+			e.printStackTrace();
+		} catch (EntityRetrievalException e) {
+			e.printStackTrace();
+		} catch (IOException e) {
+			e.printStackTrace();
+		} 
+		
+		return meaningfulUseUserResults;
+	}
+	
+	@ApiOperation(value="Get a single date value to indicate when the last meaningful use user file data is accurate as of.", 
+			notes="This is a single system-wide value.")
+	@RequestMapping(value="/accurate_as_of", method=RequestMethod.GET,
+			produces="application/json; charset=utf-8") 
+	public @ResponseBody MeaningfulUseAccurateAsOf getAccurateAsOfDate() {
+		MeaningfulUseAccurateAsOfDTO dto = muManager.getMeaningfulUseAccurateAsOf();
+		MeaningfulUseAccurateAsOf muuAccurate = new MeaningfulUseAccurateAsOf(dto);
+		if(StringUtils.isEmpty(muuAccurate.getId().toString()) && StringUtils.isEmpty(muuAccurate.getAccurateAsOfDate().toString())){
+			muuAccurate.getErrorMessages().add("Could not obtain the accurate as of date.");
+		}
+		return muuAccurate;
+	}
+	
+	@ApiOperation(value="Update the Meaningful Use Accurate As Of date.", 
+			notes="Accurate as of date value can be edited by a user with ROLE_ADMIN and ROLE_CMS_STAFF.")
+	@RequestMapping(value="/accurate_as_of", method=RequestMethod.POST,
+			produces="application/json; charset=utf-8") 
+	public @ResponseBody MeaningfulUseAccurateAsOf updateMeaningfulUseAccurateAsOf(@RequestParam(required=true) MeaningfulUseAccurateAsOf meaningfulUseAccurateAsOf) {
```

Comment:
I originally had the controller passing in a Long timeinMillis and changed it to this after thinking about it some. Here were my thoughts:
1. Designing the controller with the object passed in allows greater flexibility and forward compatibility. I added functionality to the controller that allows a user to pass in only the accurateAsOfDate, so the user doesn't need to know the id or pass that in. The user can pass in null for everything but accurateAsOfDate.
2. What if we want to update the deleted column or insert more rows in the future? 
3. What if we add more columns in the future? 
4. What if this table ends up housing more than one row?
5. I didn't find any examples of API calls in CHPL where the user passed in primitive type rather than an object, so I was trying to follow existing examples by using an object.